pkg/ui/app/containers/nodeGraphs.tsx

```diff
@@ -446,15 +521,26 @@ class NodeGraphs extends React.Component<NodeGraphsProps, {}> {
           <SummaryStat title="Total Nodes" value={this.props.nodeCount} />
           <SummaryStat title="Capacity Used" value={capacityPercent}
                        format={(v) => `${d3.format(".2f")(v)}%`}
-                       tooltip={`You are using ${Bytes(capacityUsed)} of ${Bytes(capacityTotal)} 
+                       tooltip={`You are using ${Bytes(capacityUsed)} of ${Bytes(capacityTotal)}
                        storage capacity across all nodes.`} />
+          <SummaryStat title="Unavailable ranges" value={this.props.unavailableRanges} />
+          <SummaryStat title="Queries per second" value={this.props.queriesPerSecond} />
+          <SummaryStat title="P50 latency" value={this.props.p50Latency} format={(n) => d3.format(".1f")(NanoToMilli(n)) + " ms"} />
```

Comment:
Ok I misunderstood how that reducer worked. Put some more time in and managed to switch over to your implementation.